docs/prover/cli/conf-file-api.md

```diff
@@ -0,0 +1,143 @@
+Certora Prover Conf File Format
```

Comment:
I'd consider shortening this to "Conf files".  Also the CLI Options page, now that I'm thinking about it.

Reply:
changed in both places